.gitignore

```
@@ -25,4 +25,6 @@ yarn-error.log*
 .env
 
 cypress/screenshots
-cypress/videos
\ No newline at end of file
+cypress/videos
+.yarn/
+.env-oakley
```

could make this more generic
```suggestion
.env*
!.env.example
```

---

done